.tests/integration/config.yaml

```diff
@@ -1,10 +1,10 @@
+resources: "resources.yaml"
 samples: "samples.tsv"
 units: "units.tsv"
-resources: "resources.yaml"
 
 default_container: "docker://hydragenetics/common:0.0.1"
 
 fastp_pe:
```

Comment:
If we don't trim this will not be used, should it then be removed?

Reply:
done